### PR TITLE
:sparkles: Random fixes

### PR DESCRIPTION
**Pull request by Sigmanificient**

### CODE REVIEW

File: pincer/commands/commands.py [hunk 1]
```diff
@@ -739,14 +739,15 @@ async def add_commands(self, commands: List[AppCommand]):
...
+    @staticmethod
+    def __build_local_commands():
         """Builds the commands into the format that Discord expects. See class info
         for the reasoning.
         """
         for cmd in ChatCommandHandler.register.values():
 
             if cmd.sub_group:
-                # If a command has a sub_group, it must be nested to levels deep.
+                # If a command has a sub_group, it must be nested to level deep.
```
Comment (Endercheif) [inline pincer/commands/commands.py:750]: Is this suppose to mean this?
```suggestion
                # If a command has a sub_group, it must be nested 2 levels deep.
```

File: pincer/objects/message/user_message.py [hunk 2]
```diff
@@ -508,7 +508,7 @@ async def remove_emoji(self, emoji):
             f"/channels/{self.channel_id}/messages/{self.id}/reactions/{emoji}"
         )
 
-    # TODO: Implement file (https://discord.com/developers/docs/resources/channel#edit-message)
+    # TODO: Implement file (https://discord.def/resources/channel#edit-message)
```
Comment (Endercheif) [inline pincer/objects/message/user_message.py:511]:
```suggestion
    # TODO: Implement file (https://discord.dev/resources/channel#edit-message)
```

**Review by Endercheif — CHANGES_REQUESTED** [review]

File: pincer/objects/events/presence.py [hunk 3]
```diff
@@ -199,7 +199,7 @@ class Activity(APIObject):
     secrets: APINullable[:class:`~pincer.objects.events.presence.ActivitySecrets`]
         Secrets for Rich Presence joining and spectating
     instance: APINullable[:class:`bool`]
-        "nether or not the activity is an instanced game session
+        nether or not the activity is an instanced game session
```
Comment (trag1c) [inline pincer/objects/events/presence.py:202]:
```suggestion
        whether or not the activity is an instanced game session
```

**Review by trag1c — APPROVED** [review]

**Review by Endercheif — APPROVED** [review]
🪨

**Review by Lunarmagpie — APPROVED** [review]

**Pull request merged**